automated install/basic-install.sh

```diff
@@ -1872,15 +1883,12 @@ installPihole() {
 
     # If the user wants to install the Web interface,
     if [[ "${INSTALL_WEB_INTERFACE}" == true ]]; then
-        if [[ ! -d "/var/www/html" ]]; then
+        if [[ ! -d "${webroot}" ]]; then
             # make the Web directory if necessary
-            mkdir -p /var/www/html
+            install -d -m 0755 -o ${LIGHTTPD_USER} -g ${LIGHTTPD_GROUP} ${webroot}
```

Comment:
We can not assume the user wants lighttpd installed at this point (we have not validated `INSTALL_WEB_SERVER=true`).

Reply:
Fixed.